blueman/bluez/Base.py

```diff
@@ -40,7 +40,7 @@ class Base(Gio.DBusProxy, metaclass=BaseMeta):
 
     _interface_name: str
 
-    def __init__(self, obj_path: str):
+    def __init__(self, *, obj_path: str):
```

Comment:
Why is this needed?

Reply:
`BaseMeta.__call__` expects to find `obj_path` in the keyword arguments, so if you pass it as a positional argument, the whole thing fails.